docs/typescript/patching.md

```diff
@@ -101,3 +149,5 @@ If while we're deploying `v2deprecatedpatch` (below) there are still live Worker
 
 Upgrading Workflow dependencies (such as ones installed into `node_modules`) _might_ break determinism in unpredictable ways.
 It is highly recommended to use a lock file (`package-lock.json` or `yarn.lock`) in order to fix Workflow dependency versions and gain control of when they're updated.
+
+Be aware that any of your dependencies might do something non-deterministic, such as filesystem access or network calls.
```

Comment:
This is not true because WF runs in a sandbox

Reply:
```suggestion
```